src/backend/defs/rest-api/objects/MessageV2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+type: object
+properties:
+  attachments:
+    type: array
+    items:
+      "$ref": Attachment.json
+  body: # in future, could be only a preview of body, limited to xxx chars
```

Comment:
What about a [strategy pattern](https://en.wikipedia.org/wiki/Strategy_pattern) or [decorator pattern](https://en.wikipedia.org/wiki/Decorator_pattern) for this kind of behavior ?
The `attachments` and `subject` property can be included in those patterns. 

What I mean is a communication is not only simple text. It can be raw email text content, html, markdown, visio call, audio call, encrypted text, and what next ...? 

Message must be flexible at this point.